closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -215,25 +187,36 @@ private RandomIO(File seedFile, File nextFile) throws IOException {
             }
             this.seedIn = seedStream;
             this.nextIn = nextStream;
-            this.nextBuffer = new byte[bufferSize];
         }
 
+        // get the SHA1PRNG for mixing
+        // initialize if not yet created
+        private SHA1PRNG getMixRandom() {
+            SHA1PRNG prngObj = mixRandom;
+            if (prngObj == null) {
+                synchronized (LOCK_GET_SEED) {
+                    prngObj = mixRandom;
+                    if (prngObj == null) {
+                        try {
+                            byte[] seedBytes = new byte[20];
```

Comment:
Rather than hard-coding `20` here, a reference to `SHA1PRNG.DIGEST_SIZE` would be better. Alternative changes to `SHA1PRNG` that don't require callers to know how many seed bytes are required are possible; the usage pattern might be:
```
    prngObj = SHA1PRNG.seedFrom(seedIn);
```

Reply:
done